---
title: Research in the Stajich Lab
layout: default
group: research
---

# Our research topics
## Sequencing the fungal kingdom
I am co-leading projects to sequence more than 1000 fungal genomes ([1KFG](http://1000.fungalgenomes.org/)) in order to sample evolutionary and ecological diversity of the kingdom. This is enabled through the CSP the JGI and their sequencing capacity and exploration of new technologies. My research is focused on evolution of the early branching fungi and supporting this by culturing and obtaining DNA from these lineages in collaboration with several projects including the [ZyGoLife Research Collaborative](http://zygolife.org/) and individual labs working on chytrid fungi. We are applying phylogenetic methods to these large datasets and developing tools to extract informative data from a variety of sequencing strategies. Our data and code are made available on these sites. [[stajichlab github](https://github.com/stajichlab)] [[hyphaltip github](https://github.com/hyphaltip)] [[1KFG github](https://github.com/1KFG)]

## Evolution of fungal cell wall
The cell wall protects and partitions the cellâ€™s internal components from the outside world, but is permeable allowing materials and chemical signals to be transported.  The composition of the cell wall varies across the fungal kingdom from the aquatic chytrids to the zygomycetes, and ascomycete and basidiomycete fungi. We  seek to describe the differences in cell walls comparing lineages of early branching fungi and understand the evolution of the genes that synthesize and assemble the cell wall components.  We will use enzymatic and imaging approaches to visualize and quantify cell wall composition in a variety of fungi and connect this to genome composition of cell wall biosynthesis, remodeling, and related genes via genome sequencing and comparative genomics. This work extends from our work on the early diverging chytrid fungi like Batrachochytrium dendrobatidis (Bd) and interest in how fungi evade detection from plant or animal defense and immune systems through modifications in the cell wall.

## Evolution of multicellularity
The evolution of multicellular structures in fungi is the result of many changes in cell wall structure and adhesion, gene regulation for development and tissue differentiation, and signaling pathways that allow for specific responses from differentiated tissues. How these changes evolved in the fungi from what was likely a single-cell ancestor into the multitude of forms of fungi we can observe as molds, mushrooms, smuts, rusts, lichens, and yeasts.  We hope to unravel the evolutionary process in more detail exploring the lineage-specific pathways, new aspects of gene regulation and signaling, and evolutionary changes in cell wall biosynthesis and maintence that will help delineate what is required to form multicellular structures in the extant fungi.  We are focused on flagellated Chytridiomycota and Blastocladiomycota fungi to study early changes in the fungi in addition to comparisons of independent origins of multicellularity in Taphrinamycotina, Pezizomycotina (Ascomycota) and within the Basidiomycota fungi.

## Transposons, gene duplication, and genome evolution
What role does duplication play in genome evolution?   We are interested in quantifying how much change in gene family size/membership is due to neutral evolution and what kinds of gene family size changes are driven by direction selection.  Conversely are gene families that rarely change in size under purifying selection for copy number?

How do mechanism that control transposon and retroviral element proliferation in fungal genomes impact how gene families can expand and does this impact evolutionary adaptability by duplication?  In some fungi including N. crassa there are pathways limiting duplication including Repeat Induced Point mutations ([RIP](http://fungalgenomes.org/wiki/RIP)) and meitoic silencing of unpaired DNA ([MSUD](http://fungalgenomes.org/wiki/MSUD)).  How do gene families evolve in these fungal lineages and what strategies for replication have succeeded for some transposon families?

As part of this work we are also interested in the evolution of mobile elements in fungi with highly effective genome defense mechanisms.  Our lab is investigating variations in the frequency of transposable elements and the history of genome invasion events among different fungal phyla and the interplay with genome defense evolution.

This work also links to two projects.

- One is the phenotypic impact of transposon expansion in rice which is a collaboration with the [Wessler lab](http://wesslerlab.ucr.edu/) at UCR, and [Tom Brutnell](http://bti.cornell.edu/TomBrutnell.php) at the Boyce Thompson Institute and [Qi Sun](http://cbsu.tc.cornell.edu/staff/qisun/) at Cornell and funded by the [NSF](http://www.nsf.gov/awardsearch/showAward.do?AwardNumber=1027542) ([UCR press release](http://newsroom.ucr.edu/news_item.html?action=page&id=2583)).
- The second is a project on identifying active transposons in mosquitos and is work with the [Wessler](http://wesslerlab.ucr.edu/) and [Atkinson](http://www.facultydirectory.ucr.edu/cgi-bin/pub/public_individual.pl?faculty=86) labs at UCR ([UCR press release](http://newsroom.ucr.edu/news_item.html?action=page&id=2549)).

## Methods for comparative and population genomics
Genome sequencing so affordable and accessible that nearly any fungal genome can be sequenced.  Managing the data from the currently over 100 fungal genomes and integrating new data requires disciplined software and data management approaches.  We build tools to mining these data to address research questions from phylogenetics and systematics, molecular evolution and population genetics, to predicting metabolic and enzymatic capabilities. Aspects of this work will be presented in databases useful to the community at the site we manage [http://fungalgenomes.org](http://fungalgenomes.org/) that include some genome browsers and wiki tools.

We also work in collaboration with the [EuPathDB](http://eupathdb.org/) group to build a new resource called [FungiDB](http://fungidb.org/) which enables data mining functional & comparative fungal genomic data. This project, funded by the [Burroughs Wellcome Fund](http://www.bwfund.org/), went live with a new website in March 2011.

Our lab is also engaged in analysis of whole genome resequencing data to study population genomics of some animal pathogens to better understand genome dynamics and populations, especially in emerging infections diseases such as *Bd*.

## Post-Transcriptional Gene Regulation in Fungi
Gene regulation occurs at many levels in the cell.  Understanding the players and role of transcriptional and post-transcriptional regulation controlling development in fungi will provide insight into the evolution of development. We are exploring the role of smallRNAs in fungi and how they play a role in regulation of gene expression controlling development. This work is performed in the filamentous fungus *Neurospora crassa* applying genetic and genomic tools.
